### DISCUSSION TITLE
Contributing to the repo

### DISCUSSION THREAD

**Discussion by SammygoodTunes** [phst-randomizer/ph-randomizer#434] [General]
Hi,

Is there any way I can get to know the project a little more and throw in some contributions myself?
Randomisers are cool and since the zelda ds games are basically my childhood (especially ST), I thought this could be a good first project to contribute to.

I've had a look at the contents and had a read at the couple of docs that explain how to do so, but I'd like to discuss this further privately as here wouldn't be particularly appropriate. Though, that would be up to you.

I've seen other randomisers (like OoT and MM, and BoTW iirc?) and I think a randomiser for PH would be cool (and sooner or later ST which would be even cooler).
I've also done a lot of playing around with AR codes and memory editing (and some, albeit very little, disassembly and reverse-engineering work (Ghidra) around DS games), especially for ST. I'm somewhat familiar with the libnds library as well.

I did attempt to contribute to the ph-docs repository, however I noticed that there was automation going on (commits scheduled every day?) so I ended up ceasing the fork I had made. I can provide more info about that if necessary.

Anyway, let me know if this is possible.

Thanks!

**Comment by mike8699** [comment]
Hi! Thanks for the interest. Happy to discuss more, we have a discord that we use to coordinate development - send me a message (mike100220) and I'll add you to it.
